api/v1beta1/aerospikebackupservice_webhook.go

```diff
@@ -52,6 +53,10 @@ func (r *AerospikeBackupService) Default() {
 	if r.Spec.Resources != nil && r.Spec.PodSpec.ServiceContainerSpec.Resources == nil {
 		r.Spec.PodSpec.ServiceContainerSpec.Resources = r.Spec.Resources
 	}
+
+	if r.Spec.PodSpec.ServiceAccountName == "" {
```

Comment:
Why not make it like GetAerospikeInitContainerImage func

Reply:
Moved to func call in the reconciler for now.